style.css

```diff
@@ -0,0 +1,71 @@
+html {
+   box-sizing: border-box;
+   font-size: 100%;
+}
+
+*,
+*::after,
+*::before {
+   box-sizing: inherit;
+}
+
+body {
+   padding: 0;
+   margin: 0;
+   color: white;
+   font-family: "Poppins", sans-serif;
+   background-color: #27272A;
+}
+
+/* Header styles  */
+.header {
+   height: 80px;
+   background-color: #be3144;
+   display: flex;
+   justify-content: end;
+   align-items: center;
+   margin-bottom: 0;
+}
+
+.header-nav {
+   display: flex;
+   margin-right: 26px;
+   height: 100%;
+}
+
+.header-nav > a {
+   padding-inline: 16px;
+   text-decoration: none;
+   color: white;
+   font-weight: 400;
+   font-size: 26px;
+   padding-top: 20px;
+}
+
+.header-nav > a:hover {
+   background-color: rgb(111, 111, 216);
+}
+
+.welcome{
+   height: 500px;
+   text-align: center;
+   display: flex;
+   justify-content: center;
+   align-items: center; 
+}
+
+.h1-wrapper{
+   height: min-content;
+}
+
+h1{
+   font-size: 42px;
+}
+
+.web-dev-txt{
```

Comment:
Use a proper naming scheme for classes as this will help you be more organized in your styling. An example of this is the [Block element modifier(BEM)](http://getbem.com/naming/)

Use this for all class names